docs/source/methods/depletion.rst

```diff
@@ -0,0 +1,242 @@
+.. _methods_depletion:
+
+=========
+Depletion
+=========
+
+When materials in a system are subject to irradiation over a long period of
+time, nuclides within the material will transmute due to nuclear reactions,
+producing other nuclides which may be radioactive. The time-dependent process by
```

Comment:
Can you elaborate or suggest a change? I'm not quite sure what you mean here

Reply:
On first pass, it seems like materials only transmute due to irradiation, rather than radioactive decay. That conclusion is reached only by reading this statement, however. I'm not sure the level of knowledge the reader is assumed to have in these documents, so most people may just know that some isotopes decay naturally and spontaneously in the absence of irradiation

Reply:
Gotcha -- with that in mind, when I re-read it, I see what you are saying. I'll modify this to try to make it clearer.

Reply:
Ok, hopefully it makes more sense now